boskos/client/client.go

```diff
@@ -110,7 +115,14 @@ func NewClient(owner string, url string) *Client {
 // Acquire asks boskos for a resource of certain type in certain state, and set the resource to dest state.
 // Returns the resource on success.
 func (c *Client) Acquire(rtype, state, dest string) (*common.Resource, error) {
-	r, err := c.acquire(rtype, state, dest)
+	return c.AcquireWithPriority(rtype, state, dest, "")
```

Comment:
nit: Could this DRY to a lower level so you don't need to pass the empty ID?

Reply:
I did not want to change the signature Acquire as it is used in multiple place. Do you have an example of where you would like this to be ?